react.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
 
         "react/jsx-indent-props": [2, 4],
 
-        "react/jsx-space-before-closing": 2,
+        "react/jsx-tag-spacing": 2,
```

Comment:
Yeah, I noticed that, but now we have a chance to fix it :) A separate PR is also cool, but we should wait this PR to be merged.